app/views/client/articles/_article-list.html.erb

```diff
@@ -0,0 +1,66 @@
+<div class="article-list">
+	<div class="container-large">
+		<h2 class="article-list__title">intermezzo</h2>
+			
+		<div class="article-list__item">
+			<a href="#" class="article-list__item-link">
```

Comment:
tag a nya dikasi class "article-list__item", lalu container "article-list__item" diatas dihapus, dan properti dari class "article-list __item-link" dipindah

Reply:
already change class of tag a and delete class item-link